docs/plugins.md

```diff
@@ -8,7 +8,7 @@ redirect_from:
  - /docs/usage/modules/
 ---
 
-Out of the box Babel doesn't do anything. In order to actually do anything to your code
+Out of the box Babel doesn't do anything. In order to actually do something to your code
```

Comment:
`use plugins` instead of `enable plugins`? "enable"  sounds like it's already there in babel-core but "use" means to use any plugin on npm.